tests/test_security_oauth2.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
This file looked like a good example of complex dependency usage, and allowed me to get 100% coverage for tests.

I also wanted to start by making a minimal set of changes to make the pull request easier to review; simply adding `from __future__ import annotations` seemed like an easier way to prove it works than creating an entirely new test file to review.

@tiangolo How would you propose we handle testing this functionality?
* Should I copy this file (or another) and just add the `from __future__ import annotations` import?
* Should I create an entirely new file with a more comprehensive set of string-annotation tests?
    * In this case, is there a good place for me to look for examples?